Makefile

```diff
@@ -10,7 +10,10 @@ OUTPUT	:= ./output
 TEST	:= ./test/
 TESTOUTPUT := ./test/testOutput
 
-all: piLeib piMonte mv mm fibonacci mv-openmp
+all: piLeib piMonte mv mm fibonacci mv-openmp piLeib_openmp
```

Comment:
I fixed the merge conflict here to include both your changes and the changes made in master